packages/Charts/src/demos/demo.ts

```diff
@@ -0,0 +1,71 @@
+import * as DG from 'datagrok-api/dg';
+import * as grok from 'datagrok-api/grok';
+
+
+export class Demo {
+  static async chordViewerDemo() {
+    const df = await grok.data.getDemoTable('energy_uk.csv');
+    const tableView = grok.shell.addTableView(df);
+    tableView.addViewer('ChordViewer');
+  }
+
+  static async globeViewerDemo() {
+    const df = await grok.data.getDemoTable('geo/earthquakes.csv');
+    const tableView = grok.shell.addTableView(df);
+    // semantic type detection is needed for the latitude, longitude and magnitude columns
+    DG.debounce(df.onSemanticTypeDetected, 50).subscribe((_) => tableView.addViewer('GlobeViewer'));
+  }
+
+  static async groupAnalysisViewerDemo() {
+    const df = await grok.data.getDemoTable('demog.csv');
```

Comment:
Need a better dataset (and maybe better default settings) for that. Perhaps @MariaDolotova might help.

Reply:
done